person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age:, parent_permission: true, name: 'Unknow')
+    @id = rand(1..100)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    return true if @age >= 18 || @parent_permission == true
+
+    false
+  end
```

Comment:
- [ ] There is no need to repeat ourselves here. Let's use the `of_age?` method for the first half of the conditional. The `@parent_permission` instance variable is either `true` or `false` anyway so we don't need to check if it's true again. I propose you refactor this method to this:

  ```
  def can_use_services?
    of_age? || @parent_permission
  end
  ```